docs/source/guide/build_pipelines/index.rst

```diff
@@ -184,6 +184,8 @@ Finally, we have the automatically generated Dockerfile (``demo_airflow_pipeline
 
    CMD [ "standalone" ]
 
+.. _testingairflow:
```

Comment:
its a mark so that we can use this section as a reference elsewhere. The reference still exists but the mark got lost during a prev refactor. adding it back here because sphinx shouts at me every time i build.